themes/grass/layouts/contribute/devel.html

```diff
@@ -40,11 +40,15 @@ <h3>Quick links</h3>
 
             </nav>
             <ul class="list-unstyled">
-	      <li><i class="fa fa-file-text"></i> &#160;<a href="https://grass.osgeo.org/programming7/" target="_blank">Developer manual</a></li>
-	      <li><i class="fa fa-github"></i> &#160;<a href="https://github.com/OSGeo/grass/" target="_blank"> Source code</a></li>
-	      <li><i class="fa fa-bug"></i> &#160;<a href="https://github.com/OSGeo/grass/issues" target="_blank"> Bug Tracker</a></li>
-              <li><i class="ms ms-grass-gis"></i> &#160;<a href="https://trac.osgeo.org/grass/wiki/HowToContribute" target="_blank"> How to contribute</a></li>
-            </ul>
+	            <li><i class="ms ms-grass-gis"></i> &#160;<a href="https://trac.osgeo.org/grass/wiki/HowToContribute" target="_blank">How to contribute</a></li>
+              <li><i class="fa fa-code"></i> &#160;<a href="https://github.com/OSGeo/grass/blob/master/CONTRIBUTING.md" target="_blank">How to start to code</a></li>
```

Comment:
This is more of a summary of Howtogit, not really an introduction on how to code. I would thus change the title

```suggestion
              <li><i class="fa fa-code"></i> &#160;<a href="https://github.com/OSGeo/grass/blob/master/CONTRIBUTING.md" target="_blank">How to use git to contribute</a></li>
```